src/app/lib/utilities/srcSet/index.js

```diff
@@ -69,6 +87,17 @@ export const createSrcsets = ({
 export const getPlaceholderSrcSet = ({ originCode, locator, isWebP }) => {
   if (!originCode) return '';
   if (!locator) return '';
```

Comment:
I know it's not your code but might as well tidy this up
```suggestion
  if (!originCode || !locator) return '';
```

Reply:
> I know it's not your code but might as well tidy this up

Thanks for the suggestion 👍